modules/decision/decision_worker.py

```diff
@@ -0,0 +1,75 @@
+"""
+Retrieves list of landing pads from cluster estimation and outputs decision to the flight controller 
+
+"""
+
+from utilities.workers import queue_proxy_wrapper
+from utilities.workers import worker_controller
+from . import decision
+from . import landing_pad_tracking
+from . import search_pattern
+from ..flight_interface.flight_interface_worker import current_odometry, odometry_mutex
+
+
+def decision_worker(
+    camera_fov_forwards: float,
+    camera_fov_sideways: float,
+    search_height: float,
+    search_overlap: float,
+    distance_squared_threshold: float,
+    small_adjustment: float,
+    tolerance: float,
```

Comment:
Order these by appearance of the class objects below (i.e. `distance_squared_threshold` first because `LandingPadTracking` is first).